### PR TITLE
Add support for Arch Linux

### PR DESCRIPTION
**Pull request by mmerfort**
* Add Arch Linux default params
* Do not install some packages on Arch Linux, they are included in the
  ruby package
* Add tests for Arch Linux

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 language: ruby
 
 rvm:
-  - 1.9.3
```
Comment (ghoneycutt) [inline .travis.yml]: This should not be dropped here. If it is to be dropped, it should be done separately along with changes to the README.

Comment (mmerfort) [inline .travis.yml]: I removed this commit and will make a new separate PR to remove the unsupported Puppet Versions and update the documentation.

**Comment by ghoneycutt** [comment]
Thanks @mmerfort
